orderer/common/server/util.go

```diff
@@ -27,3 +31,31 @@ func createLedgerFactory(conf *config.TopLevel, metricsProvider metrics.Provider
 	}
 	return lf, nil
 }
+
+// ValidateBootstrapBlock returns whether this block can be used as a bootstrap block.
+// A bootstrap block is a block of a system channel, and needs to have a ConsortiumsConfig.
+func ValidateBootstrapBlock(block *common.Block, bccsp bccsp.BCCSP) error {
```

Comment:
why is this still needed?

Reply:
To check that there is no system channel in chains and join blocks when the server starts... did that a few weeks ago.

Reply:
I just added a fix - that the code will be using the new ValidateBootstrapBlock instead of the one in the onboarding package.

Reply:
But do we still use the one in the onboarding package after this change? if no then we can remove it no?

Reply:
Yes we can, @tock-ibm correct me if I'm wrong. 
Do you want to do the removal of the package in the scope of this PR?

Reply:
Yes ;-)